projects/plugins/jetpack/extensions/blocks/donations/tabs.js

```diff
@@ -39,7 +39,7 @@ const Tabs = props => {
 				annualDonation: { ...annualDonation, planId: products[ '1 year' ] },
 			} ),
 		} );
-	}, [ oneTimeDonation, monthlyDonation, annualDonation, setAttributes, products ] );
+	}, [ oneTimeDonation, monthlyDonation, annualDonation, products, setAttributes ] );
```

Comment:
Do we have to include setAttributes here?! Also this change is quite uneneeded :)

Reply:
Ha! I didn't notice that.  But yeah, your PR removed it and the linter wanted it back, so I re-added it.  I'll get this fixed.

Reply:
Same, I'm not a big fan of it.